controller/konnect/sdkfactory.go

```diff
@@ -0,0 +1,36 @@
+package konnect
+
+import (
+	sdkkonnectgo "github.com/Kong/sdk-konnect-go"
+	sdkkonnectgocomp "github.com/Kong/sdk-konnect-go/models/components"
+)
+
+// SDKToken is a token used to authenticate with the Konnect SDK.
+type SDKToken string
+
+// SDKFactory is a factory for creating Konnect SDKs.
+type SDKFactory interface {
+	NewKonnectSDK(serverURL string, token SDKToken) *sdkkonnectgo.SDK
+}
+
+type sdkFactory struct {
+	ServerURL string
+	Token     SDKToken
```

Comment:
What's the point of having these fields in the factory, since they are passed to the `NewKonnectSDK` function?

Reply:
Whoops, you're right.

```suggestion
```

Reply:
Removed in 6466abce2253ee90f78c945d96d82ca53dd05745.